src/fft.h

```diff
@@ -28,7 +28,8 @@ FFT<T>::FFT(GF<T> *gf, int n)
 {
   this->gf = gf;
   this->n = n;
-  this->inv_n_mod_p = (gf->inv(n) % gf->p);
+  //this->inv_n_mod_p = (gf->inv(n) % gf->card());
```

Comment:
I think this is gf->card() here, @lamphamsy can you fix ?

Reply:
You did it :)